java/core/src/main/java/sleeper/core/properties/instance/TableStateProperty.java

```diff
@@ -176,6 +176,11 @@ private static UserDefinedInstancePropertyImpl.Builder propertyBuilder(String pr
             .description("The maximum given concurrency allowed for the transaction deletion lambda.\n" +
                     "See maximum concurrency overview at: https://aws.amazon.com/blogs/compute/introducing-maximum-concurrency-of-aws-lambda-functions-when-using-amazon-sqs-as-an-event-source/")
             .propertyGroup(InstancePropertyGroup.TABLE_STATE).build();
+    UserDefinedInstanceProperty TRANSACTION_SNAPSHOT_DELETION_LAMBDA_TIMEOUT = Index.propertyBuilder("sleeper.statestore.transactionlog.snapshot.deletion.lambda.timeout.seconds")
+            .description("The maximum timeout for the transaction deletion lambda in seconds")
```

Comment:
This could do with a full stop at the end.

Reply:
Addressed